resources/views/dashboard/admin/toggles/show.blade.php

```diff
@@ -33,10 +33,12 @@
                         @endif
                     </td>
                     <td>{{ $t->toggle_description }}</td>
-                    <td>
-                        <center>
-                            <a href="/dashboard/admin/toggles/toggle/{{ $t->toggle_name }}" class="btn btn-primary simple-tooltip" data-toggle="tooltip" title="Flip Toggle"><i class="fas fa-sliders-h"></i></a>
-                        </center>
+                    <td class="text-center">
+                        <div class="btn-group">
+                            <a href="/dashboard/admin/toggles/toggle/{{ $t->toggle_name }}" class="btn btn-primary simple-tooltip" data-toggle="tooltip" title="Flip Toggle"><i class="fas fa-sliders-h fa-fw"></i></a>
+                            <a href="/dashboard/admin/toggles/edit/{{ $t->toggle_name }}" class="btn btn-warning simple-tooltip" data-toggle="tooltip" title="Edit Toggle"><i class="fas fa-pencil-alt fa-fw"></i></i></a>
+                            <a href="/dashboard/admin/toggles/delete/{{ $t->toggle_name }}" class="btn btn-danger simple-tooltip" data-toggle="tooltip" title="Edit Toggle"><i class="fas fa-trash fa-fw"></i></i></a>
```

Comment:
This title should be "Delete Toggle"

Reply:
@iccowan thanks! That's why we QA. Fixed.